pkg/fakedata/generator.go

```diff
@@ -177,6 +178,27 @@ var enum = func(column Column) string {
 	return withEnum(enum)(column)
 }
 
+var fileCache map[string][]string
+
+var file = func(column Column) string {
```

Comment:
@lucapette is there a reason you put the functions in `var`s instead of the usual `func file() ...`?